### PR TITLE
Basic pre-submit jobs for openshift-knative/kn-plugin-event

### PR DESCRIPTION
**Pull request by cardil**
Basic pre-submit jobs for openshift-knative/kn-plugin-event

### CODE REVIEW

File: ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+binary_build_commands: ./mage build
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml:10]: There's already a stable release which we can use.

Comment (cardil) [reply ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml:10]: +1

Comment (cardil) [reply ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml:10]: I did change that to `stable`, but it failed: 

```
invalid configuration: releases.latest.stream: must be one of ci, nightly
```

Comment (cardil) [reply ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml]: I've end up using:

```
releases:
  latest:
    release:
      channel: stable
      version: "4.9"
```

File: ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+binary_build_commands: |
+  export HOME=/tmp
+  ./mage build
```
[the remark this answers is not in the document]
Comment (cardil) [reply ci-operator/config/openshift-knative/kn-plugin-event/openshift-knative-kn-plugin-event-release-0.26.yaml:3]: I've written that intentionally

**Pull request merged**